### PR TITLE
doc: reword protocol step summary

### PR DESCRIPTION
**Pull request by kerzhner**

### CODE REVIEW

**Review by rjdrost — COMMENTED** [review]
Thanks Mike, this is much clearer for the reader to understand the state of the Alice's ticket through the process. It also states clearly that Alice has the ability to withdraw her funds on the ExitChain once her ticket is included in the batch if needed (for add'l cost)

File: doc/SAFE.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ Thus, Alice's swap is serviced with `1 + 2/n` transactions on `ExitChain` and `1
...
 
-1. Alice deposits `x` tokens on `ExitChain`, and is given a ticket `t`, which records that "`x` tokens should be sent to Alice on `EntryChain`", which is initially in the `pending` state.
-2. Bob then authorizes `t` by signing a special message `b` containing a batch of tickets, moving it into the `authorized` state.
-3. Once it's authorized, Alice _has the ability_ to submit `b` to `EntryChain`, which sends `x` tokens to Alice. (Bob will probably submit `b` for expediency and convenience.) Bob is then forced to wait for a `SafetyWindow` timeout to pass
-4. After the `SafetyWindow` timeout passes, Bob can receive Alice's `x` tokens by calling `claimExitChainBatch(b)` on `ExitChain`, which moves `t` into the `claimed` state.
+1. Alice deposits `x` tokens on `ExitChain`, and is given a ticket `t`. The ticket is initially in the `pending` state and records that "`x` tokens should be sent to Alice on `EntryChain`."
+2. Bob then authorizes `t` by including the ticket in a signed batch `b`. The ticket is now in the `authorized` state.
```
Comment (NiloCK) [inline doc/SAFE.md:19]:
```suggestion
2. Bob then authorizes `t` by including it in a signed batch `b`. The ticket is now in the `authorized` state.
```

Comment (NiloCK) [inline doc/SAFE.md:19]: Maybe specify that `b` is written to the ExitChain at this point (accounting for one of Bob's ExitChain transactions).

```suggestion
2. Bob then authorizes `t` on the `ExitChain` by including the ticket in a signed batch `b` and calling `authorizeExitChainBatch(b)`. The ticket is now in the `authorized` state.
```

**Review by NiloCK — APPROVED** [review]
Definite improvement. LGTM.

File: doc/SAFE.md (hunk 1 above)
Comment (kerzhner) [reply doc/SAFE.md]: Done in 3984557.

**Pull request merged**